integration/testdata/TestTmsUploadIntegration/bin_param/mta.yaml

```diff
@@ -0,0 +1,14 @@
+_schema-version: "3.1.0"

```

Comment:
Do we have to duplicate these files? Instead you could have one dir for the testdata and set the `customConfig` flag.

```
--customConfig string         Path to the pipeline configuration file (default ".pipeline/config.yml")
```

So the config.yaml can have an atypical filename, such as `tms_integration_test_config.yaml` and then with the `customConfig` flag you set the config for the test case where you need it:
```
err := container.whenRunningPiperCommand("tmsUpload",
		"--customConfig=.pipeline/tms_integration_test_config.yaml")
```

Reply:
Thanks for the tip! Implemented